packages/api/package.json

```
@@ -15,7 +15,8 @@
     "migration:run": "node ace migration:run",
     "migration:refresh": "node ace migration:refresh",
     "seed": "node ace seed",
-    "seed:default": "node ace seed --files=\"0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js\""
+    "seed:default": "node ace seed --files=\"0_RoleSeeder.js, 2_TaxonomySeeder.js, 3_TermSeeder.js, 5_PermissionSeeder.js\"",
+    "apidoc": "./node_modules/apidoc/bin/apidoc -e \"(node_modules|public)\" -o public/apidoc"
```

```suggestion
    "apidoc": "apidoc -e \"(node_modules|public)\" -o public/apidoc"
```

Considerando que apidoc é uma dependência você não precisa colocar o `path` completo do binário

---

:+1: